src/commands/monorepo/monorepo_restore_cache.yml

```
@@ -0,0 +1,42 @@
+parameters:
+  monorepo_engine:
+    description: Monorepo engine
+    type: string
+    default: "lerna"
+  package:
+    description: Package name
+    type: string
+    default: ""
+  package_folder:
+    description: Package folder
+    type: string
+    default: "packages"
+  cache_identifier:
+    description: Package folder
+    type: string
+    default: "monorepo-build-cache"
+steps:
+  - when:
+      condition:
+        equal:
+          - all
+          - "<< parameters.package >>"
+      steps:
+        - restore_cache:
+            keys:
+              - "<< parameters.cache_identifier >>--{{ .Environment.CACHE_VERSION }}-{{ .Branch }}--{{ .Revision }}"
+              - "<< parameters.cache_identifier >>--{{ .Environment.CACHE_VERSION }}-{{ .Branch }}--"
+              - "<< parameters.cache_identifier >>--{{ .Environment.CACHE_VERSION }}-master--"
+        - when:
+            condition:
+              equal:
+                - "lerna"
+                - "<< parameters.monorepo_engine >>"
```

```suggestion
              equal: ["lerna", << parameters.release_engine >>]
```
For better readability